packages/blob/src/client.ts

```diff
@@ -145,30 +179,31 @@ export function getPayloadFromClientToken(
   return JSON.parse(decodedPayload) as DecodedClientTokenPayload;
 }
 
-export const EventTypes = {
+const EventTypes = {
   generateClientToken: 'blob.generate-client-token',
   uploadCompleted: 'blob.upload-completed',
 } as const;
 
-export interface GenerateClientTokenEvent {
+interface GenerateClientTokenEvent {
   type: (typeof EventTypes)['generateClientToken'];
   payload: { pathname: string; callbackUrl: string };
 }
-export interface BlobUploadCompletedEvent {
+interface UploadCompletedEvent {
   type: (typeof EventTypes)['uploadCompleted'];
   payload: {
+    // TODO @Fabio: is this correct? I guess it should be a PutBlobResult and not HeadBlobResult unless
+    // we're doing an extra head() call somewhere?
     blob: HeadBlobResult;
```

Comment:
I think at the time we implemented this feature the 2 interfaces were exactly the same
Now the event receives 
```
 const blob: BlobPutApiResult | BlobPutApiResultV0 =
    apiVersion < 2
      ? // we're faking previous response format for the vercel/blob module not to throw
        {
          url: encodeURI(`${baseUrlForBlobs}/${key}`),
          pathname,
          contentType,
          contentDisposition,
          uploadedAt: new Date().toISOString(),
          size: parseInt(request.headers.get('content-length') ?? '0', 10),
        }
      : {
          url: encodeURI(`${baseUrlForBlobs}/${key}`),
          pathname,
          contentType,
          contentDisposition,
        };
```
so `PutBlobResult` should be the correct one

Reply:
Updated 👍